index.js

```diff
@@ -149,11 +151,13 @@ var appServer = function(config) {
 
     self.start = function() {
         // Instantiate up the server
+        //TODO: add i18n support (i18n-node might be a good look)
```

Comment:
Nitpick: personally I don't find these TODOs useful, because we have Github issues. If you want them there, maybe add a link to the github issue opened for it?